### DISCUSSION TITLE
What actor played Bill Murray's love interest in the movie "Groundhog Day"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#306] [Galaxy Trivia] [answered]
**Hint:** AMcD

**Comment by thuongtruong1009** [comment, marked as answer]
In the movie "Groundhog Day", Bill Murray's love interest was played by actress Andie MacDowell. She portrayed the character Rita Hanson, a TV producer who accompanied Murray's character, Phil Connors, on his trip to Punxsutawney, Pennsylvania to cover the annual Groundhog Day event. Throughout the film, Phil becomes enamored with Rita and tries to win her affection as he experiences the same day over and over again.
